client/my-sites/stats/feedback/index.tsx

```diff
@@ -248,4 +249,42 @@ function StatsFeedbackController( { siteId }: FeedbackProps ) {
 	);
 }
 
-export default StatsFeedbackController;
+function logFeedbackPresentationStatus( key: string, value: boolean ) {
+	const isCalypsoLive = window.location.host === 'calypso.live';
+	const isCalypsoLocalhost = window.location.host.startsWith( 'calypso.localhost' );
+	if ( isCalypsoLive || isCalypsoLocalhost ) {
+		localStorage.setItem( key, value ? 'true' : 'false' );
+	}
+}
+
+const FEEDBACK_HIGH_TRAFFIC_SITE_THRESHOLD = 10000;
+
+function getHighTrafficThreshold() {
+	const value = Number( localStorage.getItem( 'StatsHighTrafficThreshold' ) );
+	return value !== 0 ? value : FEEDBACK_HIGH_TRAFFIC_SITE_THRESHOLD;
+}
+
+interface FeedbackPresentorProps {
+	siteId: number;
+}
+
+function StatsFeedbackPresentor( { siteId }: FeedbackPresentorProps ) {
+	const { supportCommercialUse } = useStatsPurchases( siteId );
+	const { isJetpackNotAtomic, isVip } = useSiteTypes( siteId );
+	const { data, isSuccess } = useFetchTrafficHook( siteId );
+
+	const views = data?.past_thirty_days.views ?? 0;
+	const highTrafficThreshold = useMemo( () => getHighTrafficThreshold(), [] );
+	const isHighTrafficSite = isSuccess && views > highTrafficThreshold;
+	logFeedbackPresentationStatus( 'StatsHighTrafficSite', isHighTrafficSite );
```

Comment:
I missed this one before, but I don't think we should send tracks on every rendering 🤔

Reply:
This was meant to simplify testing when the prompts were limited to Odyssey Stats. It allowed me to check a site in Calypso and see if the site would get the feedback prompts in Odyssey. Now that the code has been updated to check site type instead of environment, this isn't really needed anymore. I'll get it removed!

Reply:
Could we maybe stop shipping any dev / debugging changes to production in the future, which would cause either security or performance concerns, and sometimes very confusing.